src/scripts/modules/role-acceptances/role-acceptances.coffee

```diff
@@ -65,3 +68,12 @@ define (require) ->
         alert('You must accept the license')
 
     licenseRequired: () -> !!_.findWhere(@model.get('roles'), {hasAccepted: true})
+
+    disableOrEnableSubmit: () ->
+      submit = @$el.find('.submit')
+      roles = @model.get('roles')
+      role = _.pluck(roles, 'hasAccepted')
+      if _.contains(role, null)
+        submit.prop('disabled', true)
+      else
```

Comment:
Since we're already re-rendering the template on a role change, can this be handled in the template?